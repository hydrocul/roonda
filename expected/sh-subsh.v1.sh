#!/bin/sh

(
    echo a
    echo b
)
(
    echo a
    echo b
) | cat
if echo a > /dev/null; then
    (
        echo if-t-a
        echo if-t-b
    )
else
    (
        echo if-f-a
        echo if-f-b
    )
fi
{
    echo a
    echo b
}
{
    echo a
    echo b
} | cat
if echo a > /dev/null; then
    {
        echo if-t-a
        echo if-t-b
    }
else
    {
        echo if-f-a
        echo if-f-b
    }
fi
