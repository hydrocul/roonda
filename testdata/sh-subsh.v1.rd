sh v1

(subsh (echo "a")
       (echo "b"))

(pipe (subsh (echo "a")
             (echo "b"))
      (cat))

(if (echo "a" (> "/dev/null"))
    ((subsh (echo "if-t-a")
            (echo "if-t-b")))
    ((subsh (echo "if-f-a")
            (echo "if-f-b"))))

(group (echo "a")
       (echo "b"))

(pipe (group (echo "a")
             (echo "b"))
      (cat))

(if (echo "a" (> "/dev/null"))
    ((group (echo "if-t-a")
            (echo "if-t-b")))
    ((group (echo "if-f-a")
            (echo "if-f-b"))))

