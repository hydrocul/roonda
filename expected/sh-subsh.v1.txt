a
b
a
b
if-t-a
if-t-b
a
b
a
b
if-t-a
if-t-b
